mediaproviders/src/main/java/com/kaltura/playkit/providers/ovp/KalturaOvpMediaProvider.java

```diff
@@ -337,7 +337,7 @@ private void onEntryInfoMultiResponse(String ks, ResponseElement response, OnMed
                 }
 
             } else {
-                error = response.getError() != null ? response.getError() : ErrorElement.LoadError;
+                error = response.getError() != null ? response.getError() : ErrorElement.LoadError.message("error response in multirequest. response: " + response.getResponse());
```

Comment:
@GouravSna 
please verify if response is not null before using the getter

Reply:
@giladna , This check in line no 288, in isValidResponse() method.